### DISCUSSION TITLE
How to generate ":NvimTreeGenerateOnAttach" on Windows?

### DISCUSSION THREAD

**Discussion by Nazaroni** [nvim-tree/nvim-tree.lua#2155] [Q&A] [answered]
When I try to generate ":NvimTreeGenerateOnAttach", by default it uses "/tmp/my_on_attach.lua" file path... but I'm on Windows 10 😅.
Is it possible to generate this file on Windows?

**Comment by alex-courtis** [comment, marked as answer]
See #2157
